src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -455,11 +474,14 @@ contract EigenStrategyManager is
         ITokenStakingNode node
     ) internal view returns (uint256 stakedBalance) {
 
+        IStrategy strategy = strategies[asset];
```

Comment:
the semantics of this is everything that resides in the TokenStakingNode

including if its queued or withdrawn